libs/withdraws/src/lib/withdraw-form.tsx

```diff
@@ -98,7 +98,11 @@ export const WithdrawForm = ({
           <Controller
             control={control}
             name="asset"
-            rules={{ validate: { required } }}
+            rules={{
+              validate: {
+                required: (value: string) => !!selectedAsset || required(value),
```

Comment:
chore (blocking): the form should use a default asset if one is passed in so we shouldn't need a change here